packages/perseus/src/validation.types.ts

```diff
@@ -180,7 +179,14 @@ export type PerseusSorterUserInput = {
     changed: boolean;
 };
 
-export type PerseusTableRubric = PerseusTableWidgetOptions;
+export type PerseusTableRubric = {
+    // The number of rows to display
+    rows: number;
+    // The number of columns to display
+    columns: number;
+    // Translatable Text; A 2-dimensional array of text to populate the table with
+    answers: ReadonlyArray<ReadonlyArray<string>>;
+};
```

Comment:
I don't think the other fields are used, so might be worth a try to remove them. You'd also have to remove them from the tests. 

```suggestion
    
    // Translatable Text; A 2-dimensional array of text to populate the table with
    answers: ReadonlyArray<ReadonlyArray<string>>;
};
```